src/amp-ads/40_Experimental_Ads/Scrollbound_Slides_360_Ad.html

```diff
@@ -17,9 +17,8 @@
   <title>Scrollbound Slides 360 Ad</title>
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://amp-ads.firebaseapp.com/dist/amp-inabox.js"></script>
```

Comment:
Does this need to be replaced as well?

Reply:
I didn't replace this one and the stack-bind one because they are experimental ads and won't be valid. I didn't check with the the slides 360 ads, but for the stack-bind ad, we still need this patched version of `amp-inabox.js`. So I decided to leave those two examples unchanged. I'll revert all changes to those two examples.

Reply:
Decided to completely remove the examples first. I'll add it back in following PR after I figure out the experimental ads issue.